sdk/typescript/src/zklogin/publickey.ts

```diff
@@ -77,3 +84,84 @@ export function toZkLoginPublicIdentifier(
 	tmp.set(addressSeedBytesBigEndian, 1 + issBytes.length);
 	return new ZkLoginPublicIdentifier(tmp);
 }
+
+const VerifyZkLoginSignatureQuery = graphql(`
+	query Zklogin(
+		$bytes: Base64!
+		$signature: Base64!
+		$intent_scope: ZkLoginIntentScope!
+		$author: SuiAddress!
+	) {
+		verifyZkloginSignature(
+			bytes: $bytes
+			signature: $signature
+			intentScope: $intent_scope
+			author: $author
+		) {
+			errors
+		}
+	}
+`);
+
+export async function graphqlVerifyZkLoginSignature({
+	address,
+	bytes,
+	signature,
+	network,
+	fetch: fetchFn,
+}: {
+	address: string;
+	bytes: string;
+	signature: string;
+	network?: 'mainnet' | 'testnet';
+	fetch?: typeof fetch;
+}) {
+	const gqlClient = new SuiGraphQLClient({
+		url:
+			network === 'testnet'
+				? 'https://sui-testnet.mystenlabs.com/graphql'
+				: 'https://sui-mainnet.mystenlabs.com/graphql',
```

Comment:
So far we have avoided hardcoding endpoints into the SDK, especially in ways that can't be overwritten.  The GraphQL API is also likely to introduce path-based routing for versioned schemas, which may change these urls.  We probably need to think through the best way to handle this